package.json

```diff
@@ -133,5 +133,5 @@
     "singleQuote": true,
     "trailingComma": "es5"
   },
-  "stardog-version": ">=8.0.0"
+  "stardog-version": ">=8.2.0"
```

Comment:
we should not change this-- this package is still compatible with Stardog 8.0.0+